packages/react-docs/UPGRADE-GUIDE.md

```diff
@@ -0,0 +1,264 @@
+- **ChipGroupToolbarItem**: This component has been removed. Use `<ChipGroup categoryName="name">` instead.  All props that were on the `<ChipGroupToolbarItem>` have been added to `<ChipGroup>`. To use the new API, props to `<ChipGroup>` from `<ChipGroupToolbarItem>`. [(#4246)](https://github.com/patternfly/patternfly-react/pull/4246)
+
+
+### Context selector
+- **ContextSelectorItem:** 
+  - Removed prop `isHover`. All instances of it should be removed from your application. [(#4116)](https://github.com/patternfly/patternfly-react/pull/4116)
+- **ContextSelectorToggle:** 
+  - Removed props `isHovered` and `isFocused`. All instances of them should be removed from your application. [(#4116)](https://github.com/patternfly/patternfly-react/pull/4116)
+
+### Data list
+- Renamed `noPadding` to `hasNoPadding`. [(#4133)](https://github.com/patternfly/patternfly-react/pull/4133)
+
+### Divider
+- Removed promoted component from experimental index file. Change import path from  `import { Divider } from '@patternfly/react-core/dist/esm/experimental';` to `'import { Divider } from '@patternfly/react-core/dist/esm/components';` [(#4029)](https://github.com/patternfly/patternfly-react/pull/4029). 
+
+
+### Drawer
+- Renamed `noPadding` to `hasNoPadding`. [(#4133)](https://github.com/patternfly/patternfly-react/pull/4133)
+- Removed import path `'@patternfly/react-core/dist/js/experimental'`. The import path `'@patternfly/react-core'` should be used instead. [(#4176)](https://github.com/patternfly/patternfly-react/pull/4176)
+
+### Dropdown
+- **Dropdown:**
+  - Removed props `isHover` and `isFocus` from Toggle, KebabToggle, and DropdownToggle. All instances of them should be removed from your application. [(#4116)](https://github.com/patternfly/patternfly-react/pull/4116)
+  - Removed prop `variant`. If you were using `variant="icon"` before, use the new prop `icon` instead. [(#4147)](https://github.com/patternfly/patternfly-react/pull/4147)
+**DropdownToggle**:
+  - Removed prop `ariaHasPopup` in favor of `aria-haspopup`
+  - Renamed `iconComponent` to `toggleIndicator` [(#4038)](https://github.com/patternfly/patternfly-react/pull/4038)
+**DropdownItemIcon**:
+  - This component has been removed in favor of `<DropdownItem icon={<Icon />} />` [(#4147)](https://github.com/patternfly/patternfly-react/pull/4147)
+
+### Empty state
+- **EmptyState**
+  - Changed the default width to `full` instead of `large`.  To maintain the previous default behaviour, set the `variant` prop to large. e.g `<EmptyState variant={EmptyStateVariant.large}> ...</EmptyState>` [(#3933)](https://github.com/patternfly/patternfly-react/pull/3933) 
+  - Variant `small` has been updated to `sm` [(#3933)](https://github.com/patternfly/patternfly-react/pull/3933)
+  - Variant `large` has been updated to `lg` [(#3933)](https://github.com/patternfly/patternfly-react/pull/3933)
```

Comment:
```suggestion
  - Updated variant `large` to `lg` [(#3933)](https://github.com/patternfly/patternfly-react/pull/3933)
```

Reply:
This was actually reverted, so I'm removing the note.